### ISSUE TITLE
Photo viewer topbar

### ISSUE THREAD

**Issue by walking-octopus** [photoview/photoview#607]
From #584

**Describe the solution you'd like**
Add a topbar to the photo viewer similar to other photo management apps.

Examples:
 - ![googlePhotos](https://user-images.githubusercontent.com/46994949/142762290-640ae66f-e32d-4e5e-8e65-7a2e2a12dcb2.png)
 - ![Nextcloud](https://user-images.githubusercontent.com/46994949/142762460-5379fef9-c9ed-47c8-a70a-d8d03a2b54e9.png)
 - ![synology](https://user-images.githubusercontent.com/46994949/142762399-c0eeae43-1012-4e9d-85b3-c1c69fd6caf5.png)

**Comment by bollnh** [comment]
👍  This feature is really needed and  useful. Especially the actions include delete, share etc.  Currently, users have no way to delete their photo from website.

**Comment by sajjadalis** [comment]
+1 I really want to zoom the images or a button to display the actual size and then scroll it or move it with the mouse.

**Comment by kkovaletp** [comment]
> Currently, users have no way to delete their photo from website.

This is by design and the product architecture. Photoview is a read-only media library tool. We don't have plans to implement any functionality to modify or delete the media from the user's library.

But other than that, I like the idea of having a navigation bar with some useful buttons

**Comment by kkovaletp** [comment]
When designing a solution for this issue, please also check all mentioned issues in the thread (at this time: #548 and #978)

**Comment by bluecasita** [comment]
This feature is absolutely needed.  Would be an amazing addition to Photoview.  Understood that Photoview is not for modifying images, but a simple delete button would be very helpful for both images and folders.

**Comment by kkovaletp** [comment]
> a simple delete button would be very helpful for both images and folders

Photoview is designed as a read-only media gallery, so no upload or delete functionality is planned. You can easily use other services to do that in parallel to Photoview, which would scan and reflect the updates in the source media library (folder)

**Comment by bluecasita** [comment]
Sorry, when I said the feature was absolutely needed, I was referencing the "tool bar" idea.  Mainly I want to see the image info when in the large image view.  

Delete functionality is nice to have.  My goal was to be able to view and and delete in the same place, rather than having to flip back and forth between applications/services and then initiate a rescan.  It would just be convenient to do it in one place.
